controller/users_blackbox_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	almtoken "github.com/almighty/almighty-core/token"
 
 	"context"
+
```

Comment:
I did read it but I mostly rely on VSCode formatting my code. It definitively a good idea to import golang packages first.